Include/cpython/ceval.h

```diff
@@ -23,3 +23,7 @@ _PyEval_RequestCodeExtraIndex(freefunc f) {
 
 PyAPI_FUNC(int) _PyEval_SliceIndex(PyObject *, Py_ssize_t *);
 PyAPI_FUNC(int) _PyEval_SliceIndexNotNone(PyObject *, Py_ssize_t *);
+#ifndef Py_LIMITED_API
```

Comment:
This `#ifndef` is not necessary; this entire header is already only included `(from `Include/ceval.h` if `Py_LIMITED_API` is not defined.